recipes/libdc1394/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+
+class Libdc1394Conan(ConanFile):
+    name = "libdc1394"
+    version = "2.2.6"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = 'https://damien.douxchamps.net/ieee1394/libdc1394/'
+    description = "libdc1394 provides a complete high level API to control IEEE 1394 based cameras"
+    topics = ("conan", "ieee1394", "camera", "iidc", "dcam")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    _env_build = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_autotools(self):
+        if not self._env_build:
+            self._env_build = AutoToolsBuildEnvironment(self)
+            if self.options.shared:
+                args = ["--disable-static", "--enable-shared"]
+            else:
+                args = ["--disable-shared", "--enable-static"]
+            self._env_build.configure(args=args)
+        return self._env_build
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Windows is not supported")
```

Comment:
It looks like only Visual Studio needs special treatment:
https://sourceforge.net/p/libdc1394/code/ci/master/tree/INSTALL-windows

Reply:
it requires a driver, so I give up for now with VS :)